tests/newfeatures/env-deploy.feature

```diff
@@ -9,7 +9,7 @@ Feature: Site Deployment
 
   @vcr site_deploy
   Scenario: Deploy dev to test
-    When I run "terminus env:deploy test --site=[[test_site_name]] --sync-content --note='Deploy test'"
+    When I run "terminus env:deploy [[test_site_name]].test 'Deploy log note' --sync-content"
```

Comment:
- Rather don't change the notes since that requires recording new fixtures. 
- I agree with Adam, if it's optional it should be an option, and existing functionality has the note default to "Deploy from Terminus" so it's currently optional.